index.html

```diff
@@ -0,0 +1,212 @@
+                        </div>
+                    </div>
+                    <div class="item-container">
+                        <div class="item-container2">
+                            <div class="first"><img src="img/program_icon_05 1.png" alt=""></div>
+                            <div class="second"> <p>
+                                CC Ignite
+                            </p></div>
+                            <div class="third"> <P>get opportunities to network with CC affiliates around the world, also after the summit.</P></div>
+                        </div>
+                    </div>
+                </div>
+                <div class="button1">
+                    <button class="button1_1">Join CC Global Summit 2015</button>
+                </div>
+                <a href="" class="see-program">SEE THE WHOLE PROGRAM</a>
+            </div>
+
+        </section>
+        <section class="third-page">
+            <div class="third-container">
+                <div>
+                    <div class="second-main">
+                        <p class="speaker">Featured Speakers</p>
+                    <img src="img/title_underline.png" alt="" class="underline">
+
+                    </div>
+                    
+                </div>
+                <div class="speaker-container home-page">
+
+
+                </div>
+
+              
+                <div class="see-more">
+                        <P class="see-more-text">MORE</P>
+                        <img src="img/arrow_down 1.png" alt="">
+
+                </div>
+            </div>
+        </section>
+        <section class="about-page4">
+            <div class="page4-container">
+                <div class="second-main"> 
+                    <p class="second-page-main">Partner</p>
+                    <img src="img/title_underline.png" alt="" class="underline">
+                </div>
+            </div>
+            <div >
+                <div class="page4-container1">
+                    <img src="img/mozilla 1.png" alt="">
+                    <img src="img/googlekorea 1.png" alt="">
+                    <img src="img/naver 1.png" alt="">
+                </div>
+                <div class="page4-container2">
+                    <img src="img/daumkakaologo 1.png" alt="">
+                    <img src="img/airbnb 1.png" alt="">
+    
+                </div>
+
+            </div>
+          
+
+
+        </section>
+
+        <div class="end">
+            <div class="end1">
+                <img src="img/cck-logo1 1.png" alt="" class="end-img">
+                <div class="end-pp">
+                    <p class="end-p">2015 Creative Commons Korea. Some Rights Reserved.</p>
+                    <p class="end-p">달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.
+                        자세한 이용기능 내용과 허락조건은 http://creativecommons.org/licences/by/2.0/kr 를 보세요.</p>
+
+                </div>
+               
+            </div>
+
+        </div>
```

Comment:
- [ ] You've made excellent progress on your project! In the current implementation, the footer section is using a `<div>` tag. To adhere to semantic HTML best practices, I recommend replacing this `<div>` tag with a `<footer>` tag. Updating the tag to `<footer>` will improve the semantic structure of your code and enhance accessibility. Your attention to these details will contribute to a more well-organized and comprehensible project. Keep up the fantastic work! 🌟📝🚀